package.json

```diff
@@ -12,6 +12,7 @@
     "precoveragehtml": "npm run coverage",
     "coveralls": "nyc report --reporter=text-lcov | coveralls",
     "test": "ava test/*",
+    "preinstall": "true",
```

Comment:
is this needed? it seemed to cause problems for microsoft env

Reply:
Cannot remember why I added it 🙈  - removed it